hacks_mbof/settings.py

```diff
@@ -0,0 +1,188 @@
+# https://docs.djangoproject.com/en/1.9/howto/static-files/
+STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
+STATIC_URL = '/static/'
+
+# Extra places for collectstatic to find static files.
+STATICFILES_DIRS = (
+    os.path.join(PROJECT_ROOT, 'static'),
+)
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = getenv('DJANGO_SECRET_KEY', 'I need to be changed!')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = getenv('DJANGO_DEBUG', True)
+
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'mbof.apps.MbofConfig',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'rest_framework',
+]
+
+REST_FRAMEWORK = {
+    # FIXME: This is OK for dev., but make it safe for prod.
+    # Uncomment DEFAULT_PERMISSION_CLASSES to require username/password set by "manage.py createsuperuser"
+    # 'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAdminUser',),
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
+    'PAGE_SIZE': 10
+}
+
+MIDDLEWARE_CLASSES = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    # FIXME: Disabled security for dev. only.  Needs to be enabled and support code added for prod. use.
+    # 'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'hacks_mbof.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'hacks_mbof.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': getenv('DJANGO_DB_ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': getenv('DJANGO_DB_NAME', os.path.join(BASE_DIR, 'db.sqlite3')),
+        'USER': getenv('DJANGO_DB_USER', ''),
+        'PASSWORD': getenv('DJANGO_DB_PASSWORD', ''),
```

Comment:
Is it really safe to put a password in an environment variable?  I think it would be safer to read the password from a file, the name of which could be in an env. variable named `DJANGO_DB_PASSWORD_FILE`.

However, if you put the password in a file, it might make sense to put other attributes with it, like user, host, name, etc.

Don't use a Python file, though.  Use a text configuration file that can be easily parsed and the values sanitized before you use them.  YAML seems to be a popular choice.  You may be (probably should be) responsible for the sanitization yourself.